builder/ebssurrogate/builder_acc_test.go

```diff
@@ -0,0 +1,135 @@
+package ebssurrogate
+
+import (
+	"fmt"
+	"os/exec"
+	"testing"
+	"time"
+
+	amazon_acc "github.com/hashicorp/packer-plugin-amazon/builder/ebs/acceptance"
+	"github.com/hashicorp/packer-plugin-sdk/acctest"
+)
+
+func TestAccBuilder_EbssurrogateBasic(t *testing.T) {
+	ami := amazon_acc.AMIHelper{
+		Region: "us-east-1",
+		Name:   fmt.Sprintf("packer-plugin-amazon-ebs-basic-acc-test %d", time.Now().Unix()),
```

Comment:
Depending on how fast these test cases run there is no guarantee  that these AMI names will be different. I suggest giving them unique names to avoid any potential naming conflicts
```suggestion
		Name:   "ebsurrogate-basic-acc-test",
```